packages/webpack-config-terra/webpack.config.js

```diff
@@ -0,0 +1,250 @@
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const DuplicatePackageCheckerPlugin = require('@cerner/duplicate-package-checker-webpack-plugin');
+const aggregateTranslations = require('terra-aggregate-translations');
+const ThemePlugin = require('./lib/postcss/ThemePlugin');
+const getThemeConfig = require('./lib/utils/_getThemeConfig');
+
+/**
+ * Get the locales to be defined by the webpack build, also aggregates translations, but we should remove that in the future.
+ * @param {object} env the env object
+ * @param {string} rootPath root path of the package.
+ */
+const getLocales = ({ disableAggregateTranslations }, rootPath) => {
+  if (!disableAggregateTranslations) {
+    return aggregateTranslations({ baseDir: rootPath });
+  }
+  return undefined;
+};
+
+/**
+ * Get an array of modules that need to be resolved to during the webpack build. Should be removed after aggregate translations is removed.
+ * @param {object} env the env object
+ */
+const getResolveModules = ({ disableAggregateTranslations }) => (
+  [
+    'node_modules',
+    ...disableAggregateTranslations ? [] : ['aggregated-translations'],
+  ]
+);
+
+/**
+ * Determines the correct theme config to usee of the three options. Precedence is. ENV, options config, terra-them.config.js
+ * @param {object} options object containing the override config if provided.
+ */
+const determineThemeConfig = ({ themeConfig: overrideThemeConfig }) => {
+  const defaultTheme = process.env.THEME; // Flexes root theme for theme visual regression testing.
+
+  if (defaultTheme) {
+    return { theme: defaultTheme };
+  }
+  if (overrideThemeConfig) {
+    return overrideThemeConfig;
+  }
+
+  return getThemeConfig();
+};
+
+/**
+ * Provides the webpack dev server options to disable hot reloading. I'm going to be honest, it doesn't alway work.
+ * @param {object} env the env object containing the request to disable hot reloading.
+ */
+const getWebpackDevServerStaticOptions = ({ disableHotReloading }) => (
+  {
+    ...disableHotReloading && {
+      hot: false,
+      inline: false,
+    },
+  }
+);
+
+/**
+ * The extendable webpack config for terra.
+ * @param {object} env environment variables pass to webpack via --env.thing.
+ * @param {object} argv webpack options from the command line arguments, no custom options here.
+ * @param {object} options custom webpack options provided via a config override.
+ */
+const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
+  const processPath = process.cwd();
+  /* Get the root path of a mono-repo process call */
+  const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
```

Comment:
I don't think we need this anymore. We added this to fix running wdio from the package level... I think we've update the wdio to move to root, then scope the test run to the package dir that it was called from.